.coveragerc

```diff
@@ -10,6 +10,7 @@ omit =
     *manage*
     *0001_initial*
     */serializers.py*
+    */backends.py*
```

Comment:
Why did you eliminate this

Reply:
I eliminated this because I thought it wasn't relevant to test this as it is more of like a validation file

Reply:
What happens when you don't ignore it

Reply:
It reduces coverage